### ISSUE TITLE
[BUG] [LightGBM] [Fatal] GPU Tree Learner was not enabled in this build. Please recompile with CMake option -DUSE_GPU=1

### ISSUE THREAD

**Issue by saraswat40** [autogluon/autogluon#3246]
I am using this tutorial:
https://auto.gluon.ai/dev/tutorials/tabular/advanced/tabular-multilabel.html

I get this error on .fit():
```
[LightGBM] [Fatal] GPU Tree Learner was not enabled in this build.
Please recompile with CMake option -DUSE_GPU=1
```

I have tried to install LightGBM with prerequisites ( glibc/cmake etc ) like so:
`# pip install lightgbm --install-option=--gpu`

The installation was successful.

but the error persists.

**Comment by jtele2** [comment]
Also having this issue.

**Comment by saraswat40** [comment]
This is documented here: [](https://auto.gluon.ai/stable/tutorials/tabular/advanced/tabular-gpu.html)

I was able to work around this by starting from this docker image: https://github.com/microsoft/LightGBM/blob/master/docker/gpu/dockerfile.gpu

and then installed autogluon on it. You will also need build-python.sh from lightgbm master and run 'build-python.sh install' as it is not part of this docker image.

Also this docker image uses an old version of cuda/cudnn. I got it to work and the gpu was being used during training but I don't know if this setup is kosher.
